<ui version="4.0" >
 <class>IdentityPage</class>
 <widget class="QWidget" name="IdentityPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="KMail::IdentityListView" native="1" name="mIdentityList" />
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QPushButton" name="mButtonAdd" >
         <property name="text" >
          <string>&amp;Add...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mModifyButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>&amp;Modify...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mRenameButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>&amp;Rename</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mRemoveButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Remo&amp;ve</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mSetAsDefaultButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Set as &amp;Default</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMail::IdentityListView</class>
   <extends>QWidget</extends>
   <header>identitylistview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
